tree/ntuple/v7/inc/ROOT/RField.hxx

```diff
@@ -400,11 +400,12 @@ protected:
    std::vector<std::unique_ptr<RFieldBase>> fSubFields;
    /// Sub fields point to their mother field
    RFieldBase* fParent;
-   /// Points into fColumns.  All fields that have columns have a distinct main column. For simple fields
-   /// (float, int, ...), the principal column corresponds to the field type. For collection fields expect std::array,
-   /// the main column is the offset field.  Class fields have no column of their own.
+   /// Points to the first element of fColumns. All fields that have columns have a distinct main column.
```

Comment:
Is it always the first element, or the first column (with index 0) of the active representation?

Reply:
Changes as discussed to `fPrincipalColumn` giving access to the active representation.